src/hotspot/cpu/riscv/macroAssembler_riscv.hpp

```diff
@@ -527,12 +527,16 @@ class MacroAssembler: public Assembler {
   }
 
   // mv
-  void mv(Register Rd, address addr)          { li(Rd, (int64_t)addr); }
+  void mv(Register Rd, address addr)                  { li(Rd, (int64_t)addr); }
+  void mv(Register Rd, address addr, int32_t &offset) {
```

Comment:
Suggestion: add necessary comment for this function, maybe:
"Split address into a lower 12-bit sign-extended offset and the remainder so that the offset could be encoded in jalr or load/store instruction."

Reply:
done - added.